README.md

```diff
@@ -70,21 +69,106 @@ of 30 mins of learning "***web super powers***",
 the code is quite simple!
 
 
-### Are You _`New`_ To _Automated_ Testing?
+### Are You _`New`_ To Testing?
 
-While _this_ tutorial has ***No Pre-requisites***
+While _this_ tutorial has ***no pre-requisites***
 (_other than a computer & Chrome web browser_),
 we have written an _introductory_ step-by-step guide
-to help _anyone_ learn (_automated_) "***Test Driven Development***" (**TDD**)
-see: [github.com/dwyl/**learn-tdd**](https://github.com/dwyl/learn-tdd)
+to help _anyone_ [learn "Test Driven Development" (TDD)](https://github.com/dwyl/learn-tdd).
+
+### What about _Automated_ Testing?
 
+**Automated** testing refers to testing automated user interactions. Daydream and Nightmare automate the usual clicks and behaviours that a human would perform, so you don't have to!
 
 ## _How_?
 
 The _reason_ we have organized the tutorial with Daydream _first_
 is that you don't _need_ to know Nightmare.js to use Daydream,
 and it can be used by people without "coding" experience.
 
+### Daydream walkthrough
+
+**This tutorial assumes that you have node installed and [Nightmare installed](https://github.com/segmentio/nightmare) as a dependency**
```

Comment:
I'd be _specific_ here and write `node.js` not just "node". 
(_non-technical people or complete beginners might not know that node & node.js are used interchangeably so worth being precise_)